src/App.js

```diff
@@ -48,6 +50,12 @@ function App() {
           </>}
         </div>
       </div>
+      <div>
+      {console.log(results)}
+            {isLoaded && results && <>
+            <SongRecommendation options={results} />
+            </>}
```

Comment:
Its preferred to use `condition ? value1 : value2` style if you have more than condition.
Otherwise you may get weird behavior like rendering 0 if  `isLoaded` is true and `results` is falsy!

Reply:
I implemented that